Create-PR view empty after creating PR with empty title & description

Type: <b>Bug</b>

Create-PR view is empty after creating a PR with empty title & description using the "Create + Merge when ready" button (which doesn't do empty fields validation)

<img width="3200" alt="image" src="https://github.com/microsoft/vscode-pull-request-github/assets/16353531/cb3a3a54-d7fe-49d6-8696-4820110a69c5">


```
webview-create-pr-view-new.js:672 Error: Minified React error #301; visit https://reactjs.org/docs/error-decoder.html?invariant=301 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at oh (webview-create-pr-view-new.js:672:51920)
    at Rj (webview-create-pr-view-new.js:674:8389)
    at Qj (webview-create-pr-view-new.js:674:928)
    at Kj (webview-create-pr-view-new.js:674:831)
    at yj (webview-create-pr-view-new.js:672:88619)
    at Ig (webview-create-pr-view-new.js:672:85305)
    at bk (webview-create-pr-view-new.js:674:16511)
    at webview-create-pr-view-new.js:674:17961
    at Nj (webview-create-pr-view-new.js:672:89103)
    at ik (webview-create-pr-view-new.js:674:17947)
Ci @ webview-create-pr-view-new.js:672
webview-create-pr-view-new.js:674 Uncaught Error: Minified React error #301; visit https://reactjs.org/docs/error-decoder.html?invariant=301 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at oh (webview-create-pr-view-new.js:672:51920)
    at Rj (webview-create-pr-view-new.js:674:8389)
    at Qj (webview-create-pr-view-new.js:674:928)
    at Kj (webview-create-pr-view-new.js:674:831)
    at yj (webview-create-pr-view-new.js:672:88619)
    at Ig (webview-create-pr-view-new.js:672:85305)
    at bk (webview-create-pr-view-new.js:674:16511)
    at webview-create-pr-view-new.js:674:17961
    at Nj (webview-create-pr-view-new.js:672:89103)
    at ik (webview-create-pr-view-new.js:674:17947)
```



Extension version: 0.85.2024032704
VS Code version: Code - Insiders 1.88.0-insider (57028e3d274ebde851c7cc02123def372ca619a6, 2024-04-03T04:44:44.440Z)
OS version: Darwin arm64 23.4.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|4, 9, 8|
|Memory (System)|32.00GB (0.60GB free)|
|Process Argv|--log github.vscode-pull-request-github=debug --crash-reporter-id 3184aec0-4b31-4764-bfb0-162155975430|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
vsaa593cf:30376535
py29gd2263:30784851
c4g48928:30535728
vscrp:30624060
962ge761:30841072
pythongtdpath:30726887
welcomedialog:30812478
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30926734
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
bf62j303:30968145
cppperfnew:30980852
d34g3935:30961436
fegfb526:30952798
bg6jg535:30979844
ccp2r6:30958154
pythonait:30973460
7ghi1836:30988844
showvideoc:31003583
chatpanelt:31003590
g1icg217:30999571

```

</details>

<!-- generated by issue reporter -->

--- a/webviews/common/createContextNew.ts
+++ b/webviews/common/createContextNew.ts
@@ -185,7 +185,7 @@ export class CreatePRContextNew {
 			isValid = false;
 		}
 
-		this.updateState({ validate: true, createError: undefined });
+		this.updateState({ validate: true, createError: undefined, creating: false });
 
 		return isValid;
 	};
@@ -284,6 +284,7 @@ export class CreatePRContextNew {
 					this.updateState(defaultCreateParams, true);
 					return;
 				}
+				message.params.creating = message.params.creating ?? false;
 				message.params.pendingTitle = message.params.defaultTitle ?? this.createParams.pendingTitle;
 				message.params.pendingDescription = message.params.defaultDescription ?? this.createParams.pendingDescription;
 				message.params.baseRemote = message.params.defaultBaseRemote ?? this.createParams.baseRemote;

